Полностью рабочее приложение из 4 спринта

Daffix1 opened the pull request:
Делаю Pull-Request для того, чтобы сдать его на ревью

Daffix1 committed:
Сверстали экран, добавили шрифты (который блядские такие не работают), добавили цвета, и так по мелочи, я уже не помню, но и не важно

Daffix1 committed:
код на ревью Спринт 4

So1ovyov commented on MovieQuiz/Helpers/UIColor+Extensions.swift:
```diff
@@ -1,3 +1,10 @@
+        static var ypBackground: UIColor { UIColor(named: "YP Background") ?? UIColor.darkGray }
+        static var ypGray: UIColor { UIColor(named: "YP Gray") ?? UIColor.gray }
+        static var ypWhite: UIColor { UIColor(named: "YP White") ?? UIColor.white}
+     }
```
Сместился отступ у закрывающей скобки, давай поправим

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,154 @@
+    @IBOutlet private var noButtonOutlet: UIButton!
+    @IBOutlet private var yesButtonOutlet: UIButton!
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet private var ImageView: UIImageView!
```
Наименование констант и переменных должно быть в стиле CamelCase и начинаться с нижнего регистра.
`imageView`

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,154 @@
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet private var ImageView: UIImageView!
+    @IBOutlet private var counterLabel: UILabel!
+    @IBOutlet private var QuestionLabel: UILabel!
```
А здесь `questionLabel`

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,154 @@
     override func viewDidLoad() {
         super.viewDidLoad()
+//      настройки шрифта
+        QuestionLabel.font = UIFont(name: "YSDisplay-Medium", size: 20)
```
Эти настройки можно вынести в отдельный приватный метод `private func setupViews()`

Daffix1 committed:
Исправленый вариант кода после ревью Спринт 4
Добавил скругления у постера и рамки
Статус бар теперь белого цвета
Привел все обозначения констант и переменных в стиль CamelCase, а так же сделал command + a и control + i
Добавил блокировку кнопок, чтобы на нее не нажимали лишний раз и приложение работало корректно

So1ovyov commented:
Метод для отключения/включения кнопок:
```
    private func changeStateButtons(isEnabled: Bool) {
        yesButtonOutlet.isEnabled = isEnabled
        noButtonOutlet.isEnabled = isEnabled
    }

```

Daffix1 committed:
Блокировка кнопок, чтобы невозможно было нажать на них после первого нажатия (иначе приложение неправильно считывало нажатия)
